brownie/_cli/test.py

```diff
@@ -60,6 +61,13 @@ def main():
         elif isinstance(value, str):
             pytest_args.extend([opt, value])
 
+    if args['--coverage']:
+        # we want to load the coverage plugin
+        coverage_rc = os.path.abspath(
+            os.path.join(os.path.dirname(__file__), '.coveragerc'))
+        pytest_args.append('--cov=contracts')
```

Comment:
The main brownie `pytest` plugin is loaded in `brownie/test/plugin.py`.  I think it would be possible to [add the dependency](https://docs.pytest.org/en/latest/plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file) there, maybe during `pytest_configure`, without having to load `.coveragerc` this way?

It's also useful to move there because some users will invoke testing directly via `pytest` rather than `brownie test`.